src/hgnc/index.js

```diff
@@ -93,7 +93,7 @@ const uploadRecord = async ({
     }
 
     for (const symbol of gene.prev_symbol || []) {
-        const { sourceId, biotype } = currentRecord;
+        const { sourceId, biotype } = body;
```

Comment:
why did you change these to get from body instead? currentRecord should have done some normalization/formatting